game/graphics/pipelines/opengl.cpp

```diff
@@ -435,7 +460,9 @@ void render_game_frame(int game_width,
     options.save_screenshot = false;
     options.gpu_sync = g_gfx_data->debug_gui.should_gl_finish();
     options.borderless_windows_hacks = windows_borderless_hack;
-    if (g_gfx_data->debug_gui.get_screenshot_flag()) {
+    want_hotkey_screenshot =
+        want_hotkey_screenshot && g_gfx_data->debug_gui.screenshot_hotkey_enabled;
+    if (g_gfx_data->debug_gui.get_screenshot_flag() || want_hotkey_screenshot) {
       options.save_screenshot = true;
       options.game_res_w = g_gfx_data->debug_gui.screenshot_width;
       options.game_res_h = g_gfx_data->debug_gui.screenshot_height;
```

Comment:
Don't use this code for quick screenshots, as they need to be set using imgui which is exactly what we want to avoid.

Reply:
Okay I have separated them